ios/FluentUI/Card Nudge/CardNudgeTokens.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFCardNudgeViewStyle)
+public enum CardNudgeViewStyle: Int, CaseIterable {
+    case standard
+    case outline
+}
+
+class CardNudgeTokens: NSObject, ObservableObject {
+    @Published public var accentColor: UIColor!
+    @Published public var accentIconSize: CGFloat!
+    @Published public var accentPadding: CGFloat!
+    @Published public var backgroundBorderColor: UIColor!
+    @Published public var backgroundColor: UIColor!
+    @Published public var buttonBackgroundColor: UIColor!
+    @Published public var buttonInnerPaddingHorizontal: CGFloat!
+    @Published public var buttonInnerPaddingVertical: CGFloat!
+    @Published public var circleSize: CGFloat!
+    @Published public var cornerRadius: CGFloat!
+    @Published public var iconSize: CGFloat!
+    @Published public var innerPadding: CGFloat!
+    @Published public var interTextVerticalPadding: CGFloat!
+    @Published public var mainContentVerticalPadding: CGFloat!
+    @Published public var minimumHeight: CGFloat!
+    @Published public var outerHorizontalPadding: CGFloat!
+    @Published public var outerVerticalPadding: CGFloat!
+    @Published public var subtitleTextColor: UIColor!
+    @Published public var textColor: UIColor!
+
+    var style: CardNudgeViewStyle {
+        didSet {
+            if oldValue != style {
+                updateForCurrentTheme()
+            }
+        }
+    }
+
+    var window: UIWindow? {
+        didSet {
+            updateForCurrentTheme()
+        }
+    }
+
+    init(style: CardNudgeViewStyle) {
+        self.style = style
+
+        super.init()
+
+        // TODO: subclass MSFTokensBase
+        //self.themeAware = true
+
+        updateForCurrentTheme()
+    }
+
+    @objc open func didChangeAppearanceProxy() {
+        updateForCurrentTheme()
+    }
+
+    // TODO: subclass MSFTokensBase
+    /*override*/ func updateForCurrentTheme() {
+        // TODO: subclass MSFTokensBase
+        //let currentTheme = theme
```

Comment:
Comments intended?

Reply:
Hm good question. I had intended to originally, to make it easier to tokenize once the vNext merge completes. That said, I'm okay with scrubbing them out for now since there didn't turn out to be a _huge_ amount, if you prefer.